### ISSUE TITLE
[Bug][A11Y][Text Input] Placeholder is read even if input is filled

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#471]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

.

### OS Version

Android and iOS

### App version

0.6.0

### Describe the issue

Placeholder is read at the end while the input is already filled

<img width="412" height="129" alt="" src="https://github.com/user-attachments/assets/e494eaad-6888-4366-a317-2bd76f7c1dae" />

<img width="406" height="66" alt="" src="https://github.com/user-attachments/assets/81d856a3-cdb0-4ba4-8e02-b255859bc639" />

### Expected Behavior

Placeholder should be read as a hint only if field is empty

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (607)
Version: 1.0.0
Build type: Alpha
Build details (GitHub): 548
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (607)
Version: 1.0.0
Build type: ALPHA
Build details (GitHub): 548
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by pya35** [comment]
@nouha06 reproduced on 1.0.0 alpha (607), placeholder is still read after the text field content

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (613)
Version: 1.0.0
Build type: Alpha
Build details (GitHub): 548
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35  (accessibility team)

**Comment by nouha06** [comment]
New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (613)
Version: 1.0.0
Build type: ALPHA
Build details (GitHub): 548
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by pya35** [comment]
OK !

**Comment by B3nz01d** [comment]
ok on iOS , but @nouha06  I cannot find build 613 in android either?

**Comment by nouha06** [comment]
@B3nz01d it’s available on [App Distribution](https://console.firebase.google.com/project/ouds-f91d8/appdistribution/app/android:com.orange.ouds.alpha.flutterapp/releases)

**Comment by B3nz01d** [comment]
Finished during Sprint # 27
